src/kernel/arch/x86/interrupt_context.zig

```diff
@@ -0,0 +1,32 @@
+// Zig version: 0.4.0
+
+pub const InterruptContext = struct {
```

Comment:
This type could be moved into x86/arch.zig as I see it being used for multitasking as well.

Reply:
Moved